src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/DiscoverVariantsFromContigAlignmentsSAMSpark.java

```diff
@@ -50,17 +50,52 @@
 import static org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection.DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection.GAPPED_ALIGNMENT_BREAK_DEFAULT_SENSITIVITY;
 
 /**
- * This tool takes a SAM file containing the alignments of assembled contigs or long reads to the reference
+ * Parse aligned contigs and call structural variants.
+ *
+ * <p>This tool takes a file containing the alignments of assembled contigs or long reads
  * and searches it for split alignments indicating the presence of structural variations. To do so the tool parses
  * primary and supplementary alignments; secondary alignments are ignored. To be considered valid evidence of an SV,
  * two alignments from the same contig must have mapping quality 60, and both alignments must have length greater than
- * or equal to minAlignmentLength.
+ * or equal to min-alignment-length.</p>
```

Comment:
I believe this tool has expanded to perform other functionalities as well, and is not reflected in the original doc (e.g. some performance evaluation, call imprecise variants not based solely on alignment signatures).